packages/react/src/components/molecules/HamburgerNav/index.js

```diff
@@ -249,7 +250,7 @@ const HamburgerNav = ({
       toggleMenu
     }}
     >
-      <nav className="ma__header__hamburger__nav" aria-label="main navigation" id="hamburger-main-navigation" role="navigation">
+      <nav className="ma__header__hamburger__nav" aria-label={(headerType === 'mixed' && !isMobileWindow) ? 'utility options' : 'main navigation'} id="hamburger-main-navigation">
```

Comment:
@ygannett, can you please update this label to match your proposed label?

Reply:
https://github.com/massgov/mayflower/pull/1743/files